packages/admin-panel/src/layout/navigation/NavPanel.jsx

```diff
@@ -70,28 +77,73 @@ const Container = styled.div`
   justify-content: space-between;
 `;
 
-export const NavPanel = ({ links, user, logo, homeLink, userLinks, basePath }) => {
+const Drawer = styled(BaseDrawer)`
+  .MuiDrawer-paper {
+    width: ${props => (props.open ? NAV_PANEL_OPEN_WIDTH : NAV_PANEL_CLOSED_WIDTH)};
+    transition: width 0.2s;
+  }
+`;
+
+const HeaderContainer = styled.div`
+  display: flex;
+  overflow: hidden;
+  a {
+    overflow: hidden;
+  }
+  img {
+    width: 7rem;
+    max-width: unset;
+  }
+`;
+
+const NavListItem = ({ link, basePath, navPanelOpen }) => {
+  const ItemWrapper = navPanelOpen
+    ? React.Fragment
+    : ({ children }) => <Tooltip title={link.label}>{children}</Tooltip>;
```

Comment:
🤯

I’ve always used an if/else or ternary for this. Linters might complain about a useless fragment, but this is so much nicer to read—I’m stealing this one 👏